File: twoSum.py
from typing import List

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(len(nums) - 1):
            for j in range(i + 1, len(nums)):

                sum = nums[i] + nums[j]
                if sum == target:
                    return [i, j]

        return []
            

sol = Solution()
print(sol.twoSum([1, 2, 3, 4], 6))

File: twoSum.js
var twoSum = function(nums, target) {
    let init = 1
    for (let i = 0; i < nums.length; i++){
        for (let j = init; j< nums.length; j++){
            if (nums[i] + nums[j] == target){
                var answer = new Array();
                answer[0] = i
                answer[1] = j
                return answer
            }
        }
        init++;
    }
    return 0;
};


arr = [3, 2, 3]
tar = 6
console.log(twoSum(arr, tar))